modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/VerifyBackupPartitionsTaskV2.java

```diff
@@ -629,6 +620,8 @@ private boolean isCacheMatchFilter(DynamicCacheDescriptor desc) {
 
             completionCntr.incrementAndGet();
 
+            idleCheck.run();
```

Comment:
If I understand correctly executing method body is not needed if idleCheck fails. Consider moving it to the start of the method(not forgetting about completionCntr logic)